browser/module.go

```diff
@@ -18,19 +19,25 @@ import (
 	"github.com/grafana/xk6-browser/common"
 	"github.com/grafana/xk6-browser/env"
 	"github.com/grafana/xk6-browser/k6ext"
-	"github.com/grafana/xk6-browser/storage"
 
 	k6modules "go.k6.io/k6/js/modules"
 )
 
 type (
+	// FilePersister is the type that all file persisters must implement. It's job is
```

Comment:
`filePersister` instead of `F`?

Reply:
Fixed in 0af81728c03aa50c0b398a03ef78ae6358b616b1